gpii/node_modules/packagekit/nodepackagekit/nodepackagekit_test.js

```diff
@@ -155,6 +155,17 @@ var fluid = require("universal"),
         return matchPkg;
     };
 
+    pkTests.testPerformActionFailure = function() {
```

Comment:
@the-t-in-rtf That was not directed at you, specifically.  And, you are correct that we don't want to lint all the third-party material.  But, `gpii.js`, `index.js`, and possibly `package.json` are GPII specific, and they aren't being linted (and `gpii.js` shows lint errors).
However, this is a tangent to *this* pull request.  I'll shut up now, and raise an appropriate JIRA for it.